drivers/wasapi/audio_driver_wasapi.cpp

```diff
@@ -129,15 +76,14 @@ static bool default_input_device_changed = false;
 
 class CMMNotificationClient : public IMMNotificationClient {
 	LONG _cRef = 1;
-	IMMDeviceEnumerator *_pEnumerator = nullptr;
+	AudioDriverWASAPI *driver;
 
 public:
 	CMMNotificationClient() {}
-	virtual ~CMMNotificationClient() {
-		if ((_pEnumerator) != nullptr) {
-			(_pEnumerator)->Release();
-			(_pEnumerator) = nullptr;
-		}
+	virtual ~CMMNotificationClient() {}
+
+	void set_driver(AudioDriverWASAPI *d) {
```

Comment:
```suggestion
	void set_driver(AudioDriverWASAPI *p_driver) {
```
Prefix arguments with `p_` and use descriptive names